pkg/ccl/importccl/read_import_pgdump.go

```diff
@@ -680,17 +684,18 @@ func (m *pgDumpReader) readFile(
 				}
 				switch row := row.(type) {
 				case copyData:
-					if expected, got := len(conv.VisibleCols), len(row); expected != got {
+					if expected, got := len(i.Columns), len(row); expected != got {
```

Comment:
should this be `len(conv.IsTargetCol)` that we compare to `len(row)`? or maybe `len(targetColMapInd)` ?

Reply:
done